### ISSUE TITLE
What is the expected way to add haxelib dependencies to a project?

### ISSUE THREAD

**Issue by hubol** [HaxeFoundation/intellij-haxe#1017]
Currently my project is plagued with complaints of unresolved symbols:
![image](https://user-images.githubusercontent.com/25911974/93656326-25820780-f9ef-11ea-981b-b78b9d698d51.png)

Did I miss something in the documentation for this?

Thanks!!!

I am running IntelliJ CE 2020.1.4 with plugin 1.3.1

**Comment by EricBishton** [comment]
TL;DR: You have to set them up yourself in File->Project Structure...->Libraries dialog.

The plugin tries to pick up global haxelibs that are installed and add them to the project.  However, that doesn't always work as expected.  If you are using local haxelibs, then the plugin doesn't handle those at all.  And Haxe's project files and IntelliJ's do not synchronize with each other.  (Yes, it's on the list of things to do. ;)

In the File->Project Structure... dialog, pick the Libraries line to get the pane that you see here:
![image](https://user-images.githubusercontent.com/12093217/93657126-a8ea2b80-f9e4-11ea-942b-d117847763ec.png)

Press the "+" at the top of the center panel, and select the directory that the haxelib is installed into, including the version number as you see above.  Sometimes you have to go a bit deeper, as you see here with openfl:
![image](https://user-images.githubusercontent.com/12093217/93657157-f9618900-f9e4-11ea-9787-da55cbd94c8a.png)

The trick is that you need to pick the directory with haxelib.json in it.  Then the libraries should begin to work.

**Comment by hubol** [comment]
Hi, thank you so much for the response!

When I choose the "+" at the top of the center panel I get these options:
![image](https://user-images.githubusercontent.com/25911974/93811773-47a99e80-fc16-11ea-8eb0-e1f1bbeff1e0.png)

Which one do I pick?

**Comment by EricBishton** [comment]
If you have the plugin downloaded and enabled, you should see a Haxe library option: 
![image](https://user-images.githubusercontent.com/12093217/93835439-10e28100-fc34-11ea-95ea-07d8580d89eb.png)

**Comment by hubol** [comment]
Indeed! I never created my project as a Haxe project! Now everything is great. Thank you so much for your patience!

**Comment by EricBishton** [comment]
You're quite welcome.
